framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Deployment.java

```diff
@@ -74,4 +74,11 @@
      */
     boolean isReady();
 
+    /**
+     * This method set a router timeout for the cloud deployment.
+     *
+     * @param timeoutValue Timeout value is set as string, because router
+     * timeout is usualy configured as systen property.
+     */
+    void setRouterTimeout(String timeoutValue);
```

Comment:
Such time values could be specified using java.time.Duration .
That way you clearly specify expected format and time units for this method.